lib/seahorn/HornifyFunction.cc

```diff
@@ -732,6 +739,23 @@ void LargeHornifyFunction::runOnFunction(Function &F) {
     m_db.addRule(allVars, boolop::limp(pre, post));
   }
 
+  // Generates synthesis rules for each call to sea.synth.assert.
+  s.reset();
+  for (auto partial : getPartialFnsToSynth(F)) {
+    for (auto &cp : cpg) {
+      auto &srcBB = cp.bb();
+      if (reached.count(&srcBB) <= 0)
+        continue;
+
+      auto &targetBB = (*partial->getParent());
+      for (auto *edge : filterCpEdgesByBB(cp, targetBB)) {
+        assert(mkEdgeSynthRules(ls, *partial, *edge, targetBB, vcgen, s));
```

Comment:
`assert` is not compiled in  Release mode. Is the intention that `mkEdgeSynthRules` is used during `Debug` mode only?